Makefile

```diff
@@ -36,6 +36,9 @@ EXTRAS_FEATURES =  --no-default-features --features extras,min
 
 DENY = -D warnings -D future-incompatible -D unused -D rust_2018_idioms -D nonstandard_style
 
+STATIC_FLAG = -C target-feature=+crt-static
+STATIC_TARGET = x86_64-unknown-linux-gnu
```

Comment:
Could you explain the purpose of the ```STATIC_TARGET``` constant? The Makefile has a separate ```TARGET``` environment variable which we use to set the target argument as necessary.

Reply:
`STATIC_TARGET` is used so that cargo can build on a different target name. Essentially I thought it was easier to apply a specific target inside the makefile instead of passing a different `TARGET` to the makefile before its execution. If I try to compile without changing the target name I get this (known) cargo error: 

```
error: cannot produce proc-macro for async-trait v0.1.59 as the target x86_64-unknown-linux-gnu` does not support these crate types
make: *** [Makefile:230: build-udev-utils] Error 101`. 
```

Which is solved by forcing the target name.